chunks/lang-node/chunk.yaml

```diff
@@ -2,6 +2,6 @@ variants:
   - name: "16"
```

Comment:
do we need to keep version 16 around @princerachit ?

Reply:
Yes, `gitpod/workspace-full` and `gitpod/workspace-node-lts` include Node.js 16 LTS.

- https://hub.docker.com/r/gitpod/workspace-full
  - https://github.com/gitpod-io/workspace-images/blob/a0c5d980d7bff728256499841da27db9346c6b75/dazzle.yaml#L23-L31
- https://hub.docker.com/r/gitpod/workspace-node-lts
https://github.com/gitpod-io/workspace-images/blob/a0c5d980d7bff728256499841da27db9346c6b75/dazzle.yaml#L59-L63

Reply:
Reference for Node.js release cycle per version: https://nodejs.org/en/about/releases/